frontend/javascripts/oxalis/view/action-bar/toolbar_view.tsx

```diff
@@ -503,8 +504,9 @@ function ChangeBrushSizeButton() {
 export default function ToolbarView() {
   const hasVolume = useSelector((state: OxalisState) => state.tracing.volumes.length > 0);
   const hasSkeleton = useSelector((state: OxalisState) => state.tracing.skeleton != null);
-  const viewMode = useSelector((state: OxalisState) => state.temporaryConfiguration.viewMode);
```

Comment:
so, the view mode is not taken into account anymore? doesn't this have implications on the arbitrary mode?

Reply:
Ah I wanted to add a comment in the PR for this, but forgot. This is a clean up, because this check is not needed any more. The ToolbarView instantiation looks like this:
`{!isReadOnly && constants.MODES_PLANE.indexOf(viewMode) > -1 ? <ToolbarView /> : null}`

At some point, we decided to not render the toolbar in arbitrary modes. This check wasn't removed at the time.